<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstansiForm</class>
 <widget class="QWidget" name="InstansiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Instansi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size:20px; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Manajemen Data Instansi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelUserAktif">
     <property name="text">
      <string>User Aktif: -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNamaInstansi">
       <property name="text">
        <string>Nama Instansi</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_nama_instansi"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAlamat">
       <property name="text">
        <string>Alamat</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="input_alamat"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelKontak">
       <property name="text">
        <string>Kontak</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="input_kontak"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAdmin">
       <property name="text">
        <string>Admin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="combo_admin"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbah">
       <property name="text">
        <string>Ubah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHapus">
       <property name="text">
        <string>Hapus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBersih">
       <property name="text">
        <string>Bersih</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_cari">
     <property name="placeholderText">
      <string>Cari berdasarkan nama instansi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableInstansi">
     <column>
      <property name="text">
       <string>ID Instansi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama Instansi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alamat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kontak</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Admin</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
